src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -320,7 +320,7 @@ private SourceRepository GetSource(string source)
 
                     foreach (var resource in metadataResources)
                     {
-                        tasks.Add(resource.GetVersions(dep.Id, token));
+                        tasks.Add(resource.GetVersions(dep.Id, Logging.NullLogger.Instance, token));
```

Comment:
Lets not use NullLogger no more. We do have complaints about this thing failing without errors